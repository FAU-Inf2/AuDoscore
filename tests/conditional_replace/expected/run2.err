{ "vanilla" : 
[
{"exercises":[{"name":"ConditionalReplace","tests":[{"desc":"testEmpty","fromSecret":false,"id":"testEmpty","success":true}]}]}
,
{"exercises":[{"name":"ConditionalReplace","tests":[{"desc":"testSecret","error":"AssertionError()","fromSecret":true,"id":"testSecret","success":false}]}]}
]
, "replaced" : 
[
]
}
