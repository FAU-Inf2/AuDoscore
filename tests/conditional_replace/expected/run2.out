echo "{ \"vanilla\" : " 1>&2
echo "[" 1>&2
/bin/dash ./single_execution.sh
make[1]: Verzeichnis „/media/DATA/Arbeit/AuDoscore/tests/conditional_replace/test.8107“ wird betreten
java -XX:-OmitStackTraceInFastThrow -Xmx1024m -cp lib/json-simple-1.1.1.jar:lib/junit.jar:lib/hamcrest-core.jar:lib/junitpoints.jar:. -Djson=yes org.junit.runner.JUnitCore UnitTest || echo
JUnit version 4.12
.
Time: 0.025

OK (1 test)

make[1]: Verzeichnis „/media/DATA/Arbeit/AuDoscore/tests/conditional_replace/test.8107“ wird verlassen
.E
Time: 0.04
There was 1 failure:
1) testSecret(SecretTest)
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at SecretTest.testSecret(SecretTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:52)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

FAILURES!!!
Tests run: 1,  Failures: 1

echo "]" 1>&2
echo ", \"replaced\" : " 1>&2
/bin/dash ./loop.sh
echo "}" 1>&2
